### PR TITLE
porn [summited to mypdns.org]

### PR DESCRIPTION
**Pull request by Rexadev**

### CODE REVIEW

File: porn [hunk 1]
```diff
@@ -367,3 +367,15 @@
 0.0.0.0 yamidas.yamiboard.com
 0.0.0.0 yamigama.com
 0.0.0.0 yespornplease.com
+0.0.0.0 newtumbl.com
+0.0.0.0 tumblrpornvideos.com
+0.0.0.0 pillowfort.social
```
Comment (spirillen) [inline porn:372]: https://mypdns.org/my-privacy-dns/porn-records/-/issues/154

File: porn [hunk 2]
```diff
@@ -367,3 +367,15 @@
...
+https://www.pillowfort.social/search/porn%20gif
+0.0.0.0 switter.at
+#Mastadon instance for porn
+#Social Media that have lots of porn V
+0.0.0.0 reddit.com
+#Don't ask for example
+0.0.0.0 twitter.com
+#https://twitter.com/yPornTV
+0.0.0.0 tumblr.com
```
Comment (spirillen) [inline porn:381]: Seriously??

Comment (Rexadev) [reply porn:381]: Yeah twitter is filled with porn worse than PH

Comment (spirillen) [reply porn:381]: It was tumblr  Yes I know teitter and tumblr is full of it, but twitter, google and a likes would require a rather tight lists, as you can't block that on DNS level.

tumbler however you can block on a subdomain level.

```
porn-records$ git grep '.tumblr.com' submit_here/
submit_here/adult.mypdns.cloud/domains.list:00dirtyone.tumblr.com
submit_here/adult.mypdns.cloud/domains.list:10000-in-place.tumblr.com
submit_here/adult.mypdns.cloud/domains.list:11inchesbuffed.tumblr.com
submit_here/adult.mypdns.cloud/domains.list:123avalon.tumblr.com
submit_here/adult.mypdns.cloud/domains.list:15gts.tumblr.com
submit_here/adult.mypdns.cloud/domains.list:1991corvette.tumblr.com
submit_here/adult.mypdns.cloud/domains.list:1c3c01d.tumblr.com
submit_here/adult.mypdns.cloud/domains.list:1-last-fight.tumblr.com
submit_here/adult.mypdns.cloud/domains.list:1wete.tumblr.com
submit_here/adult.mypdns.cloud/domains.list:2girls1gif.tumblr.com
submit_here/adult.mypdns.cloud/domains.list:2knowu.tumblr.com
submit_here/adult.mypdns.cloud/domains.list:2slavesmaster.tumblr.com
submit_here/adult.mypdns.cloud/domains.list:2wjk.tumblr.com
submit_here/adult.mypdns.cloud/domains.list:3puma.tumblr.com
submit_here/adult.mypdns.cloud/domains.list:3xoticbarbii3.tumblr.com
submit_here/adult.mypdns.cloud/domains.list:4everh0rny.tumblr.com
submit_here/adult.mypdns.cloud/domains.list:4pepee.tumblr.com
submit_here/adult.mypdns.cloud/domains.list:640020.tumblr.com
submit_here/adult.mypdns.cloud/domains.list:7andgoingstrong.tumblr.com
submit_here/adult.mypdns.cloud/domains.list:80s-touch.tumblr.com
submit_here/adult.mypdns.cloud/domains.list:a737slave.tumblr.com
submit_here/adult.mypdns.cloud/domains.list:ab-blake.tumblr.com
submit_here/adult.mypdns.cloud/domains.list:abdl86.tumblr.com
submit_here/adult.mypdns.cloud/domains.list:abdlaggregator.tumblr.com
submit_here/adult.mypdns.cloud/domains.list:abdlau.tumblr.com
submit_here/adult.mypdns.cloud/domains.list:abdlbondage.tumblr.com
submit_here/adult.mypdns.cloud/domains.list:abdlchrissy.tumblr.com
submit_here/adult.mypdns.cloud/domains.list:abdldad.tumblr.com
submit_here/adult.mypdns.cloud/domains.list:abdldiaperblog.tumblr.com
submit_here/adult.mypdns.cloud/domains.list:abdlgirls.tumblr.com
submit_here/adult.mypdns.cloud/domains.list:abdlmommy.tumblr.com
submit_here/adult.mypdns.cloud/domains.list:abdlpics.tumblr.com
submit_here/adult.mypdns.cloud/domains.list:abdlpluto.tumblr.com
submit_here/adult.mypdns.cloud/domains.list:abgirlanddaddy.tumblr.com
:
```

Should not be blacklisted from domain level, only the individual subs.

**Pull request closed without merging**